File: models/arcano_maior.py
from carta import Carta

class ArcanoMaior(Carta):
    def __init__(self, nome, numero, palavra_chave, jornada, arquetipo):
        super().__init__(nome=nome, numero=numero, palavra_chave=palavra_chave)
        self.jornada = jornada
        self.arquetipo = arquetipo

File: frontend/src/model/arcanoMaior.ts
import { Carta } from "./carta";

export class ArcanoMaior extends Carta {
    jornada: string;
    arquetipo: string;

    constructor(idCarta: number, nome: string, numero: number, palavraChave: string, jornada: string, arquetipo: string) {
        super(idCarta, nome, numero, palavraChave);
        this.jornada = jornada;
        this.arquetipo = arquetipo;
    }
}